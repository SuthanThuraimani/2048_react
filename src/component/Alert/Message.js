import React from 'react';

export default function Message(props) {
    return <div className="msg" >{props.msg}</div>
}
