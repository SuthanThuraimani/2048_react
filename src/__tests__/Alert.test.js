import React from 'react';
import { shallow } from 'enzyme';
import Message from './../component/Alert/Message';

describe('Message', () => {
    it('should render correctly', () => {
        const component = shallow(<Message msg="success" />);
        expect(component).toMatchSnapshot();
    });
});