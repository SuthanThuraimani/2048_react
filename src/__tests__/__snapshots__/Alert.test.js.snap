// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Board should render correctly 1`] = `
<div
  className="msg"
/>
`;

exports[`Message should render correctly 1`] = `
<div
  className="msg"
>
  success
</div>
`;
